CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 **********
 
+[5.3.1] - 2023-08-08
+********************
+Fixed
+=====
+* Pinned openedx-events to 8.5.0 for EventsMetadata serialization
```

Comment:
Probably should clarify that this is not an exact version pin. Suggestion:

```suggestion
* Constrain openedx-events version to at least 8.5.0 (for EventsMetadata serialization support)
```

I'd also be tempted to mention something about "don't install 5.3.0" but we don't really have a process around that, so maybe not.